supereeg/model.py

```diff
@@ -341,7 +341,7 @@ def predict(self, bo, nearest_neighbor=False, match_threshold='auto',
             loc_labels[~_count_overlapping(bor.get_locs(), mo.get_locs())] = ['reconstructed']
             recon_loc = mo.locs
         else:
-            recon_loc = mo.get_locs().iloc[recon_loc_inds[0]]
+            recon_loc = np.atleast_2d(mo.get_locs().iloc[~_count_overlapping(bor.get_locs(), mo.get_locs())].iloc[recon_loc_inds[0]].as_matrix())
```

Comment:
`.as_matrix` is being depreciated-- i think the new version is `.values()`

Reply:
note: `.values()` is only supported in python 3.x (i think)...we're seeing a similar thing in hypertools.  if you don't use `.values()` then `.as_matrix()` outputs a warning message

Reply:
just a quick clarification, i believe its just `df.values` (a field, not a function): https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.values.html

Reply:
I was going off of [these instructions](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.as_matrix.html)

Reply:
How about:
```
v = list(map(int, pd.__version__.split('.')))
df = np.atleast_2d(mo.get_locs().iloc[~_count_overlapping(bor.get_locs(), mo.get_locs())].iloc[recon_loc_inds[0]]
if v[1] >= 23:
  recon_loc = df.values()
else:
  recon_loc = df.as_matrix())
```

Reply:
(this replaces the current line 344)

Reply:
ah i see, i think it may be a typo in their documentation...this throws an error for me (pandas 0.23.4):
```
df = pd.DataFrame([1, 1])
df.values()
```
but this works:
```
df = pd.DataFrame([1, 1])
df.values
#or
df.get_values()
```

Reply:
oh, interesting.  i guess just use `values` instead of `values()` then?  (i.e. what you said above 😸)

Reply:
i think so! 😁